package/src/components/Message/MessageSimple/utils/parseLinks.ts

```diff
@@ -8,7 +8,8 @@
  * */
 const emailUserName = '[\\w+\\.~$_-]+';
 const schema = `(\\w{2,15}:\\/\\/)`;
-const domain = `((?:\\w+\\.\\w+)+(?:[^:\\/\\s]+))`;
+// something.tld OR 123.123.123.123
```

Comment:
ipv4 addition 👍 👏 

how about adding ipv6 as well? its a huge regex though according to https://stackoverflow.com/a/69685444

Reply:
In my view, not necessary yet

Reply:
I can't imagine many people would add ipv6 to a chat message to be completely honest, so I decided not to worry about it. I can make the changes for it if you think it's worth, but IMHO it seems like it'd not be used in practice